File: src/SOP8/Base/Account/Base.Account.IBLL/Response/ResponseUserList.cs
using System.Collections.Generic;
using Response;

namespace Base.Account.IBLL.Response
{
    using Models;

    public class ResponseUserList : PagingMessageResult
    {
        private List<AccountUser> m_users = new List<AccountUser>();

        public List<AccountUser> Users
        {
            get { return m_users; }
            set { m_users = value; }
        }

        public ResponseUserList()
            : base()
        {
        }

        public ResponseUserList(bool success, string message)
            : base(success, message)
        {
        }
    }
}

File: src/SOP8/광양/EgovLogin/base-account-ibll/src/main/java/egovframework/base/account/ibll/response/ResponseUserList.java
package egovframework.base.account.ibll.response;

import java.util.ArrayList;
import java.util.List;
import egovframework.base.account.ibll.models.AccountUser;
import egovframework.base.response.PagingMessageResult;

public class ResponseUserList extends PagingMessageResult {

    private List<AccountUser> users = new ArrayList<>();

    public List<AccountUser> getUsers() {
        return users;
    }

    public void setUsers(List<AccountUser> users) {
        this.users = users;
    }

    public ResponseUserList() {
        super();
    }

    public ResponseUserList(boolean success, String message) {
        super(success, message);
    }
}